csrankings-b.csv

```diff
@@ -225,6 +225,7 @@ Ben Stock,CISPA Helmholtz Center,https://people.cispa.io/ben.stock,_hDa-5wAAAAJ
 Ben Swift,Australian National University,https://cs.anu.edu.au/people/ben-swift,OQdYgLEAAAAJ
 Ben W. Reichardt,University of Southern California,http://www-bcf.usc.edu/~breichar,BEGRf_EAAAAJ
 Ben Y. Zhao,University of Chicago,http://people.cs.uchicago.edu/~ravenben,cYReSuEAAAAJ
+Benedikt Ahrens,TU Delft,https://benediktahrens.gitlab.io/,hdr4g6UAAAAJ

```

Comment:
Clarify appointment level at Delft - web page indicates dual appointment.

Reply:
Benedikt is Assistant Prof.  (full time) at TU Delft, see: https://benediktahrens.gitlab.io/cv/
The other is just a secondary affiliation.